src/main/java/com/gregtechceu/gtceu/data/recipe/misc/CraftingRecipeLoader.java

```diff
@@ -88,6 +88,7 @@ public static void init(Consumer<FinishedRecipe> provider) {
 
         VanillaRecipeHelper.addSmeltingRecipe(provider, "wrought_iron_nugget", ChemicalHelper.getTag(nugget, Iron),
                 ChemicalHelper.get(nugget, WroughtIron));
+        VanillaRecipeHelper.addShapelessRecipe(provider, "iron_nugget_sawing", new ItemStack(Items.IRON_NUGGET), new ItemStack(Items.IRON_INGOT), 's');
```

Comment:
add this as part of material processing handler, NOT just for wrought iron

Reply:
Resolved in dev talks